oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/oidc/authentication/OidcAuthorizationCodeAuthenticationProvider.java

```diff
@@ -154,6 +160,22 @@ public Authentication authenticate(Authentication authentication) throws Authent
 		}
 		OidcIdToken idToken = createOidcToken(clientRegistration, accessTokenResponse);
 
+		String requestNonce = authorizationRequest.getAttribute(OidcParameterNames.NONCE);
```

Comment:
Looks like this same logic was missed in `OidcAuthorizationCodeReactiveAuthenticationManager`

Reply:
Indeed! Done now, thanks for catching that! 👍